kustomize/base/deployment.yaml

```diff
@@ -0,0 +1,18 @@
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: install-scripts
+spec:
+  template:
+    metadata:
+      labels:
+        app: install-scripts
+    spec:
+      containers:
+        - name: install-scripts
+          image: replicated/install-scripts
+          imagePullPolicy: IfNotPresent
```

Comment:
why not just add env here and overlay dev?

Reply:
It's a design decision, what's the right answer?  I'm torn on this, what should the default behavior be when there's no overlay set?  dev i think.